samples/SampleOrigin/wwwroot/Index.html

```diff
@@ -1,89 +0,0 @@
-﻿<!DOCTYPE html>
-<html>
-<head>
-    <meta charset="utf-8" />
-    <style>
-        p {
-            font-size: 20px;
-        }
-
-        .button {
-            border: none;
-            color: white;
-            padding: 10px 15px;
-            text-align: center;
-            text-decoration: none;
-            display: inline-block;
-            font-size: 14px;
-            margin: 5px 5px;
-            cursor: pointer;
-        }
-
-        .green {
-            background-color: #4CAF50;
-        }
-
-        .red {
-            background-color: indianred;
-        }
-
-        .gray {
-            background-color: gray;
-        }
-    </style>
-    <title>CORS Sample</title>
-</head>
-<body>
-    <script type="text/javascript">
-        // Make the CORS request.
-        function makeCORSRequest(method, headerName, headerValue) {
-            // Destination server with CORS enabled.
-            var url = 'http://destination.example.com:5000/';
-            var request = new XMLHttpRequest();
```

Comment:
We should also test the fetch API as I believe there are differences in the behavior in this area

Reply:
Nevermind